index.html

```diff
@@ -163,138 +158,141 @@
-            <svg class="fill-current text-black" xmlns="http://www.w3.org/2000/svg" width="18" height="18"
-              viewBox="0 0 18 18">
+            </div>
+
+            <!-- Add margin if you want to see some of the overlay behind the modal-->
+            <div class="modal-content py-4 text-left px-6">
+                <!--Title-->
+                <div class="flex justify-between items-center pb-3">
+                    <p class="text-2xl font-bold">This vendor offers:</p>
+                    <div class="modal-close cursor-pointer z-50">
+                        <svg class="fill-current text-black" xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18">
               <path
                 d="M14.53 4.53l-1.06-1.06L9 7.94 4.53 3.47 3.47 4.53 7.94 9l-4.47 4.47 1.06 1.06L9 10.06l4.47 4.47 1.06-1.06L10.06 9z">
               </path>
             </svg>
-          </div>
-        </div>
+                    </div>
+                </div>
+
+                <!--Body-->
+                <p>Menu here</p>
+                <p>content: price</p>
+                <p>content: price</p>
+                <p>content: price</p>
+
+                <!--Footer-->
+                <div class="flex justify-end pt-2">
+                    <button class="px-4 bg-transparent p-3 rounded-lg text-indigo-500 hover:bg-gray-100 hover:text-indigo-400 mr-2" onclick="alert('eatpoop')">Action</button>
+                    <button class="modal-close px-4 bg-indigo-500 p-3 rounded-lg text-white hover:bg-indigo-400">Close</button>
+                </div>
 
-        <!--Body-->
-        <p>Menu here</p>
-        <p>content: price</p>
-        <p>content: price</p>
-        <p>content: price</p>
-
-        <!--Footer-->
-        <div class="flex justify-end pt-2">
-          <button
-            class="px-4 bg-transparent p-3 rounded-lg text-indigo-500 hover:bg-gray-100 hover:text-indigo-400 mr-2"
-            onclick="alert('eatpoop')">Action</button>
-          <button class="modal-close px-4 bg-indigo-500 p-3 rounded-lg text-white hover:bg-indigo-400">Close</button>
+            </div>
         </div>
+    </div>
+    <!-- Modal End -->
+
+
+    <!-- Question Model -->
+    <div class="m-5 shadow-lg bottom-0 right-0 fixed z-50">
+        <button id="show" class="bg-red-300 hover:bg-red-500 text-red-700 font-semibold hover:text-white py-2 px-4 border border-red-500 hover:border-transparent rounded" type="button" style="transition: all .15s ease" onclick="work()">
+          +
+        </button>
+    </div>
 
-      </div>
+    <div id="mainCovid" class="iframe-container hidden">
+        <iframe id="covid" class="z-50"></iframe>
     </div>
-  </div>
-  <!-- Modal End -->
+    <div class="hidden opacity-25 fixed inset-0 z-40 bg-black" id="backdrop"></div>
+    <!-- Question model ends -->
 
-  <script src="./js/main.js"></script>
+    <script src="./js/main.js"></script>
 
 </body>
 
-</html>
+</html>
```

Comment:
Fix this file and only add new changes in the existing file instead of copying everything.
If you have an auto code formatter, turn it off